src/config/graphical-session/xorg.md

```diff
@@ -82,6 +96,25 @@ Section "InputClass"
 EndSection
 ```
 
+### Forcing the modesetting driver
+
+Create the file `/etc/X11/xorg.conf.d/10-modesetting.conf`:
+
+```
+Section "OutputClass"
```

Comment:
What do you mean with 'portable to everything'? I've chosen `OutputClass` instead of `Device` because it supports all options you can pass to `Device`, plus you can use `MatchDriver` to discriminate between GPUs based on the kernel driver, which is useful on multi-gpu setups.
But either is fine for me in this case

Reply:
On multi-GPU setups you will want an explicit set of Devices plus a Screen attached to the default, as Xorg does not deal reliably with them, often it will fail to start when you have multiple GPUs and leave it for autoconfig.